recipes/mmtf-python/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "mmtf-python" %}
+{% set version = "1.0.4" %}
+{% set sha256 = "8aa92bc66b055f952c88f2dc33cd5dd52e39df376bd8ad88cca1cc344b2990da" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - msgpack-python
```

Comment:
`msgpack-python` does not need to be included in the build requirements (due to the `--single-version-externally-managed` flag) since it's not imported in the setup.py script.